tests/test_collection_get.py

```
@@ -131,6 +131,287 @@ def _insert_test_data(self, client, collection_name: str):
+
+            # $nin should keep disjoint/null/empty, exclude overlap; missing is excluded by JSON behavior
+            result = collection.get(
+                where={"tags": {"$nin": ["ml", "python"]}},
+                include=["ids"],
+            )
+            assert result and "ids" in result
+            assert set(result["ids"]) == {"id_disjoint", "id_null", "id_empty"}
+        finally:
+            try:
+                client.delete_collection(name=collection_name)
+            except Exception as cleanup_error:  # pragma: no cover
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)